tests/README.md

```diff
@@ -56,10 +56,23 @@ pytest with other tests because they have to be run in a Dockerfile:
 - test_s3_to_glue_lambda.py
 - test_setup_external_storage.py
 
-Example)
+
+#### Running tests for lambda
 Run the following command from the repo root to run tests for the lambda function (in develop).
-You can run this locally or inside the docker image.
 
 ```shell script
 python3 -m pytest tests/test_s3_to_glue_lambda.py -v
 ```
+
+#### Running tests for setup external storage
+Run the following command from the repo root to run the integration test for the setup external storage script to check that the STS
+access has been set for a given synapse folder (in develop).
+
+This test takes in two command line arguments:
```

Comment:
When running the entire test suite, do we also need to provide these two command line arguments?

Reply:
Yes, if we wanted to run all of the tests. However, our section regarding general running all of the tests needs to be updated because we currently can't run all of our tests under the same docker image nor all of our tests locally. They need separate docker images, which is why I didn't modify this section so this should be handled when we implement the tests into our CI/CD.